app/assets/javascripts/protocols.js.coffee

```diff
@@ -70,6 +78,9 @@ $ ->
         bootstrapTable('hideColumn', 'percent_subsidy').
         bootstrapTable('hideColumn', 'subsidy_committed')
 
+    $(document).on 'click', '#coordinator-menu', (e) ->
```

Comment:
Is this still required after removing the `link_to`s?

Reply:
Yes, this was still needed to prevent being taking into the protocol instead of just showing the coordinators dropdown.